lectures/heavy_tails.md

```diff
@@ -1212,6 +1284,8 @@ ax.violinplot(data)
 plt.show()
 ```
 
++++ {"user_expressions": []}
```

Comment:
@chappiewuzefan please remove this markup that has been added by `jupytext`

Reply:
Sure, this is super strange, is there a way not automatically add this line 😢

Reply:
Sadly not. But you can now open the `md` file directly in `jupyter lab` if you use `jupyterlab-myst` to get around this conversion issue.